src/arch/arm/armv/armv7-a/cache.c

```diff
@@ -10,11 +10,6 @@
 
 #include <arch/machine/hardware.h>
 
-static inline void invalidateByWSL(word_t wsl)
```

Comment:
Is set_irq_active also considered part of an interface? In that case I'll close this pull request, otherwise I'll update this request or create a new one with only set_irq_active.